app/components/FoodCatagory/index.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Typography,
+} from "@mui/material";
+
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { THEME_COLORS } from "@/constants";
+import { ItemFrontend } from "@/types/delivery";
+import FoodItem from "../FoodItem";
+
+interface FoodCatagoryProps {
```

Comment:
Since the context is clear "FoodCategory file and component", consider calling it props. Same for other component props.

Reply:
Solved